### ISSUE TITLE
Build error while compiling for production with Angular CLI and 1.0.0-rc.4

### ISSUE THREAD

**Issue by Gustav0ar** [Teradata/covalent#1111]
Hi,

I'm getting build errors when building for production the app using angular cli with: ng build --prod
It compiles fine if I compile with JIT.

**ERROR in : Unexpected value 'CovalentLoadingModule in /node_modules/@covalent/core/loading/loading.module.d.ts' imported by the module 'CoreModule in /src/app/core/core.module.ts'. Please add a @NgModule annotation.**

The above error shows the error with CovalentLoadingModule but the bellow code shows with the CovalentLayoutModule. It shows the error with the first package declared in imports in NgModule...
I tried falling back to importing all packages in '@covalent/core' path but it didn't work either.

```
import { CovalentLayoutModule } from '@covalent/core/layout';
import { CovalentStepsModule  } from '@covalent/core/steps';
import { CovalentLoadingModule } from '@covalent/core/loading';

@NgModule({
  imports: [
    CovalentLayoutModule,
    CovalentStepsModule,
    CovalentLoadingModule,
...
  ],
  declarations: [
...
  ],
  exports: [
...
  ],
  providers: [
...
  ]
})
export class CoreModule { }
```

Angular Version: 5.2.3
Angular CLI Version: 1.6.7
Angular Material Version: 5.1.1
Covalent Version: 1.0.0-rc.4

**Comment by aciccarello** [comment]
Related to #1099

**Comment by Gustav0ar** [comment]
Ok, I closed the issue, seems to be resolved when using TS 2.6.2
